composer.json

```
@@ -11,13 +11,13 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^8.2",
         "doctrine/orm": "^2.5.3",
         "friendsofsymfony/rest-bundle": "^2.5 || ^3.0",
         "jms/serializer-bundle": "^3.0 || ^4.0 || ^5.0",
         "php-task/php-task": "^1.3.3 || ^2.0",
         "php-task/task-bundle": "^2.0 || ^3.0",
-        "sulu/sulu": "^2.4.0 || ^2.5.0@dev",
+        "sulu/sulu": "^2.4.0 || ^2.5.0@dev || ^2.6",
```

2.4 would require still supported `7.2` in this bundle, as 2.4 and 2.5 is EOL we can go with:

```suggestion
        "sulu/sulu": "^2.6 || ^2.6@dev",
```